src/Microsoft.VisualStudio.Editor.Razor/DefaultVisualStudioDocumentTracker.cs

```diff
@@ -192,6 +201,12 @@ public void Unsubscribe()
         {
             _foregroundDispatcher.AssertForegroundThread();
 
+            if (!_subscribed)
+            {
+                return;
+            }
```

Comment:
If you're going to go this route don't you need some kind of refcount? Presumably whomever creates the tracker will un-create it.

Reply:
Hu? You'll have to explain more. Not sure I see how this would need a refcount of any sort.